build/travis-deploy.sh

```diff
@@ -10,37 +12,33 @@ if [ "${TRAVIS_EVENT_TYPE}" = "pull_request" ]; then
       exit 0
     fi
     echo "Building/pushing PR$TRAVIS_PULL_REQUEST from $PR_USER"
-    make push IMG_TAG="PR$TRAVIS_PULL_REQUEST" GOARCH=amd64
-    make clean IMG_TAG="PR$TRAVIS_PULL_REQUEST" GOARCH=amd64
-    make push IMG_TAG="PR$TRAVIS_PULL_REQUEST" GOARCH=arm64
-    make clean IMG_TAG="PR$TRAVIS_PULL_REQUEST" GOARCH=arm64
-    make push IMG_TAG="PR$TRAVIS_PULL_REQUEST" GOARCH=arm
-    make clean IMG_TAG="PR$TRAVIS_PULL_REQUEST" GOARCH=arm
-    make push IMG_TAG="PR$TRAVIS_PULL_REQUEST" GOARCH=s390x
-    make clean IMG_TAG="PR$TRAVIS_PULL_REQUEST" GOARCH=s390x
-    make push IMG_TAG="PR$TRAVIS_PULL_REQUEST" GOARCH=ppc64le
-    make clean IMG_TAG="PR$TRAVIS_PULL_REQUEST" GOARCH=ppc64le
+    for GOARCH in ${GOARCHES[@]}; do
```

Comment:
You should surround in quotes when using `@` style expansion in loops (doesn't technically matter in this case as there are no special characters, but doing so now will future proof it and prevent possible errors down the road
```suggestion
    for GOARCH in "${GOARCHES[@]}"; do
```